content/getting-started/getting-started.md

```diff
@@ -1,25 +1,50 @@
 ---
-description: Running the first build on Codemagic without any configuration
-title: Running the first build
+description: How to set up your projects to build, test and publish them with Codemagic
+title: Setting up projects
 weight: 2
 ---
 
-Upon login, Codemagic will automatically display the list of apps in your repository. 
+Codemagic allows you to build, test and publish Flutter apps (Android, iOS, web, macOS, Linux), native Android and iOS apps, native apps containing Flutter modules, Flutter widgets and Flutter packages. Projects can be configured via GIU (Flutter projects only) or using [`codemagic.yaml`](../building/yaml) (all projects).
+
+Upon login, Codemagic will automatically display the list of apps in the [connected repositories](./signup#connecting-several-repository-accounts). If you want to build an app that is not available through the account you signed up with, you can [add the app from custom source](./adding-apps-from-custom-sources).
+
+{{< figure size="" src="../uploads/2019/07/app_dashboard.PNG" caption="Codemagic Applications page" >}}
 
 {{% notebox %}}
-If you can't see your app listed, it may be because you don't have sufficient permissions or Codemagic has no access to your team or organization. Codemagic requires read/write permission to build your app. Contact your repository admin to review the settings.
+* If you can't see your app listed, it may be because you don't have sufficient permissions or Codemagic has no access to your team or organization. Codemagic requires read/write permission to build your app. Contact your repository admin to review the settings.
+* If your app requires accessing private Git submodules or dependencies, you need to give Codemagic access to them in order to build successfully. See how to do that [here](../building/access-private-git-submodules). 
+
 {{% /notebox %}}
 
-Select a Flutter app and start the very first build with preconfigured defaults by clicking **Start your first build**.
+## Running the first build
 
-{{< figure size="" src="../uploads/2019/07/app_dashboard.PNG" caption="Codemagic Applications page" >}}
+An app is not fully set up before you run the first build. During the build, Codemagic will fetch app sources, list the branches in it, look for tests and create webhooks for automatic building (for apps added via OAuth). If your repository contains multiple projects, you can select the right project after the first build, read more about it [below](#multiple-projects-in-one-repository).
```

Comment:
Some details are incorrect. We fetch app sources during every build - nothing special about that. Additionally, we sync branches and create webhooks during app creation.
Checking for tests (for prefilling the test arguments for Flutter) and listing projects is indeed done during the first build. And sure, fetching app sources is a requirement for these, which is why we do not do these tasks during app creation. So feel free to keep that in.

Reply:
Would you then say that the app is already created before the first build? That was my aim here actually - to describe the app creation process. I assumed the app is created during the initial build and not before.

Reply:
An app is indeed created before the first build. Only thing the first build does is prefill test arguments and scan for multiple projects so you can choose them for the second build.